Content.Shared/Roles/StartingGearPrototype.cs

```diff
@@ -36,10 +37,23 @@ public class StartingGearPrototype : IPrototype
         [DataField("id", required: true)]
         public string ID { get; } = default!;
 
-        public string GetGear(Slots slot, HumanoidCharacterProfile? profile)
+        public IReadOnlyDictionary<string, string> UniformAlternates => _uniformAlternates;
+        [DataField("uniformalternates")]
```

Comment:
```suggestion
        [DataField("uniformAlternates")]
```

Rest of yaml would also need fixing.